### PR TITLE
Fix lag banner

### PR DESCRIPTION
**Pull request by benheng**
When a user dismisses the lag warning/error, it won't reappear for another hour and only if it's still lagging, or until they refresh the app.

screen shot 2018-02-27 at 16 44 26

### CODE REVIEW

File: SingularityUI/app/components/common/Application.jsx [hunk 1]
```diff
@@ -1,37 +1,73 @@
...
-  const taskLagMinutes = props.maxTaskLag / 1000 / 60;
-  if (taskLagMinutes >= 3) {
-    Messenger().error({
-      message: `
-        <strong>Singularity is experiencing some delays.</strong> 
-        The team has already been notified. (Max task lag: ${Utils.duration(props.maxTaskLag)})
-      `
-    });
+const PAUSE_INTERVAL_IN_MS = 1000 * 60 * 60;
+const LAG_THRESHOLD_IN_MS = 1000 * 60 * 3;
```
Comment (andyhuang91) [inline SingularityUI/app/components/common/Application.jsx]: `DISMISS_TASK_LAG_NOFICATION_DURATION_IN_MS`
`MAX_TASK_LAG_NOTIFICATION_THRESHOLD_IN_MS`

File: SingularityUI/app/components/common/Application.jsx [hunk 2]
```diff
@@ -1,37 +1,73 @@
...
-    </div>
-  );
-};
+  componentWillUnmount() {
+    clearInterval(this.timerId);
+  }
+
+  pauseNotification() {
+    this.setState({ canShowLagError: false });
+    this.timerId = setInterval(() => {
```
Comment (andyhuang91) [inline SingularityUI/app/components/common/Application.jsx]: This should be setTimeout since it only needs to happen once

Comment (andyhuang91) [inline SingularityUI/app/components/common/Application.jsx]: `reenableTaskLagNotificationTimeoutId`

**Comment by kwm4385** [comment]
🚢

**Pull request merged**